corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -19,7 +19,64 @@
             <option value="">{% trans "Text" %}</option>
             <option value="barcode_scan">{% trans "Barcode" %}</option>
+            <option value="fixture">{% trans "Lookup Table Selection" %}</option>
           </select>
+          <button type="button"
+                  class="btn btn-default"
+                  data-bind="
+                        visible: property.appearance() == 'fixture',
+                        attr: {'data-target': '#lookup-table-prompt-' + property.uniqueId}"
+                  data-toggle="modal">
+            <i class="fa fa-pencil"></i>
+            {% trans "Edit" %}
+          </button>
+          <span data-bind="visible: property.itemSet.nodeset, text: property.itemSet.nodeset"></span>
+          <div class="modal fade" data-bind="attr: {'id': 'lookup-table-prompt-' + property.uniqueId}">
+            <div class="modal-dialog modal-md">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                  <h3>
+                    {% trans "Add Lookup Table" %}
+                  </h3>
+                </div>
+                <div class="modal-body">
+                  <div class="form-horizontal">
+                    <div class="form-group">
+                      <label class="control-label">
+                        {% trans "Lookup Table" %}
+                      </label>
+                      <select data-bind="value: property.itemSet.nodeset" id="lookup-table-select">
+                        <option value="">{% trans "Select a lookup table..." %}</option>
+                        {% for item in  js_options.item_lists %}
+                          <option value="instance('{{ item.id }}'){{ item.path }}">{{ item.name }}</option>
+                        {% endfor %}
+                      </select>
+                    </div>
+                    <div class="form-group">
+                      <label class="control-label">{% trans "Label Field" %}</label>
+                      <input class="form-control" type="text" data-bind="value: property.itemSet.label"/>
+                    </div>
+                    <div class="form-group">
+                      <label class="control-label">{% trans "Value Field" %}</label>
+                      <input class="form-control" type="text" data-bind="value: property.itemSet.value"/>
+                    </div>
+                    <div class="form-group">
+                      <label class="control-label">{% trans "Sort Field" %}</label>
+                      <input class="form-control" type="text" data-bind="value: property.itemSet.sort"/>
+                    </div>
+                  </div>
+
+                </div>
+                <div class="modal-footer">
+                  <button class="btn btn-primary" type="button" data-dismiss="modal">
+                    {% trans "Save" %}
```

Comment:
Nitpick: how about "Close" or "Done" since clicking this doesn't save the module?

Reply:
:+1: